### ISSUE TITLE
Make the list of clients more manageable

### ISSUE THREAD

**Issue by Giszmo** [nostorg/clients#6]
I had been working on a list of nostr clients, too, for nostr.info and wonder if we could work together and consolidate the data.

I asked @xrviv to break your lengthy csv into individual files which makes it easier to add and edit clients. His #5 is quickly getting ready to merge without regressions in the UI although he might take another week as he's going afk until Sunday now.

If you are interested, we could maybe further refine the format and throw our lists together. Mine is [here](https://github.com/Giszmo/nostrResources/tree/master/data).

I'm missing an explicit license to your project. Mine is MIT. I think, splitting the pure list from the UI details makes sense and I'm happy to help you migrate your findings to my repo and integrate it in your UI.

I'm also not attached to my repository but I would hope to drive consolidation in the "lets collect all information on all those nostr clients that are popping up left and right" space so if you want to break out your data into your own such data repo, I would also support that, provided you don't oppose adding details I care about.

**Comment by badonyx** [comment]
I've added a license and merged #5 with some changes. This was something that I had wanted to do but needed the nudge so thank you both for that.

**Comment by Giszmo** [comment]
I would get rid of the order.yml and add some preference int to the individual clients. Maybe assign numbers between 1 and 100 and sort by that number and name. Whenever you feel there is a wrong order, you can boost one or the other client but else, adding a client is done with just one file addition.

**Comment by badonyx** [comment]
I was not able to find a way to do that within the limitations of github pages.

**Comment by Giszmo** [comment]
It should be straight forward, no?

Assuming you add a field "score",

```
{% assign sorted = site.clients | sort: 'score' | reverse %}
```

should work.

**Comment by badonyx** [comment]
No, because `site.data.clients` is a Hash and not a collection of objects with properties. You can write a ruby plugin to sort like this but that's not supported on github pages.

https://stackoverflow.com/a/33215732

**Comment by Giszmo** [comment]
Does github pages allow you to do custom collections? I created many custom collections in my Jekyll project but I don't use github pages. If so, I can help you turn clients into a collection ... if you want that.

**Comment by badonyx** [comment]
I don't know. There might be some other solution with jekyll 4.0 but github pages is still on jekyll 3.9.

Anyway this is overcomplicating things. The explicit list is simple and works without adding UI details back into the client data.

**Comment by Giszmo** [comment]
:) I might have something for you later on. I'm working on a nostr based opinions plugin that would allow you to sort the list after pulling opinions from nostr itself. You would control who are esteemed opinion-havers to avoid spam.
